### ISSUE TITLE
Icons are too small

### ISSUE THREAD

**Issue by pmbanka** [erixtekila/gitbook-plugin-richquotes#21]
When using Gitbook v2.6.5, the default icon size looks like this:

![image](https://cloud.githubusercontent.com/assets/9840334/12223443/e8295586-b7d7-11e5-8e4d-1e4d8a4fafd8.png)

This is because css rule for `fa-2x` `font-size` get overriden by something from the gitbook itself (sorry for not being more specific, I can't speak css).

The workaround is adding a custom css rule:

```
blockquote.alert > strong {
    font-size: 2em !important;
}
```

Note: this is very similar to https://github.com/rlmv/gitbook-plugin-anchors/issues/8

**Comment by erixtekila** [comment]
This is related to the new gitbook css priorities.
See there also https://github.com/erixtekila/gitbook-plugin-richquotes/issues/15

**Comment by roidelapluie** [comment]
Indeed, this is very annoying

**Comment by rlmv** [comment]
This is now fixed in version 0.6.0 of gitbook-plugin-anchors.

**Comment by erixtekila** [comment]
Is related to #15
